src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import duke.command.*;
+
+import java.time.LocalDate;
+
+/**
+ * A parser class which parses the user input to make it understandable to the duke program.
+ */
+public class Parser {
+    private static final String DELIMIT = " ";
+    private static final String ESCAPE = "/";
+
+    /**
+     * A static method which parses the user input and returns a command based on the input.
+     *
+     * @param input User input.
+     * @return Command based on the user input.
+     * @throws DukeException Errors in the user input.
+     */
+    public static Command parse(String input) throws DukeException {
+        String[] inputs = input.split(" ", 2);
+        String head = inputs[0].toUpperCase();
```

Comment:
head seems to be ambiguous name. One suggestion would be to change it to commandType